### PR TITLE
Fixed Documentation. Updated output and references to files generated…

### PR DESCRIPTION
**Pull request by IzakMarais**
… when running gocc -v

### CODE REVIEW

File: doc/gocc_user_guide.tex [hunk 1]
```diff
@@ -750,25 +776,15 @@ \section{Example: Using an AST} \label{sec:example ast}
...
-		=== RUN TestFail
+		--- PASS: TestPass (0.00s)
+		=== RUN   TestFail
 		input: a b ; d e f
-		--- FAIL: TestFail (0.00 seconds)
-		ast_test.go:23: 	Error: illegal -1(-1) ; @ 1:5, expected one of: id $
-		        FAIL
-		exit status 1
-		FAIL	github.com/goccmack/gocc/example/astx	0.015s	\end{verbatim}
+		--- PASS: TestFail (0.00s)
```
Comment (awalterschulze) [inline doc/gocc_user_guide.tex]: Could we maybe this test still log its output so that it corresponds to the text in two paragraphs from this point.

Comment (awalterschulze) [inline doc/gocc_user_guide.tex]: Or do you know why the error is no longer happening?

**Comment by awalterschulze** [comment]
The rest looks good.

File: doc/gocc_user_guide.tex (hunk 1 above)
Comment (IzakMarais) [inline doc/gocc_user_guide.tex]: The test itself has been updated to no longer fail and print an error, but rather to expect an error:
https://github.com/goccmack/gocc/blob/master/example/astx/ast_test.go#L20-L25

Comment (awalterschulze) [inline doc/gocc_user_guide.tex]: Ok great.  Can we make that test do a t.Log with the error and then reflect that output in the docs.  Maybe that will help the docs make more sense, what do you think?

Comment (IzakMarais) [inline doc/gocc_user_guide.tex]: Will update the test as well to print out the error message.

**Comment by IzakMarais** [comment]
Yes, my thoughs exactly. Will add it now.  

    On Tuesday, May 30, 2017 8:55 AM, Walter Schulze <notifications@github.com> wrote:
 

 @awalterschulze commented on this pull request.In doc/gocc_user_guide.tex:>  		input: a b ; d e f
-		--- FAIL: TestFail (0.00 seconds)
-		ast_test.go:23: 	Error: illegal -1(-1) ; @ 1:5, expected one of: id $
-		        FAIL
-		exit status 1
-		FAIL	github.com/goccmack/gocc/example/astx	0.015s	\end{verbatim}
+		--- PASS: TestFail (0.00s)
Ok great. Can we make that test do a t.Log with the error and then reflect that output in the docs. Maybe that will help the docs make more sense, what do you think?—

File: doc/gocc_user_guide.tex (hunk 1 above)
Comment (awalterschulze) [inline doc/gocc_user_guide.tex]: awesome :)

**Comment by awalterschulze** [comment]
Looks good to me.

@mewmew or @goccmack you want to second that?  Before I merge?

**Comment by goccmack** [comment]
I haven't followed and am not in a position to do so shortly. I trust your judgement.

**Comment by mewmew** [comment]
LGTM. Feel free to merge.

**Comment by awalterschulze** [comment]
Thank you @IzakMarais good work and good catch :)

**Pull request merged**